src/less/chip/stories/static-chip.stories.js

```diff
@@ -0,0 +1,73 @@
+export default { title: "Skin/Chip/Static Chip" };
+
+export const typical = () => `
+<span class="chip">
+    <span class="chip__text">Football</span>
+</span>
+`;
+
+export const leastContent = () => `
+<span class="chip">
+    <span class="chip__text">I</span>
+</span>
+`;
+
+export const mostContent = () => `
+<span class="chip">
+    <span class="chip__text">Chip with the longest potentially possible content</span>
+</span>
+`;
+
+export const set = () => `
```

Comment:
I like this. Might be nice to have a story with a basic inline list too? Just to make sure no styles from chip cause any issue with that. 

e.g.

```
<ul style="list-style: none; margin: 0; padding: 0">
    <li style="display: inline">
        <span class="chip">
            <span class="chip__text">Football</span>
        </span>
    </li>
    ...
</ul>
```

Reply:
I'm not sure what use case that would cover, but I'll add it regardless.

Reply:
If we can't think of a use case, then no need. But I was considering things that might upset the inline layout of a list.

Reply:
Already added locally. 😄 I'll just keep it.